app/routers/endpoints.py

```diff
@@ -1,52 +1,81 @@
-from fastapi import APIRouter, Depends, HTTPException
+import os
+from dotenv import load_dotenv
+from fastapi import APIRouter, Depends
 from sqlalchemy.orm import Session
 from app import crud, schemas
 from app.database import get_db
-from app.models import models
+from app.utils.logger_config import setup_logger
+import logging
 
 router = APIRouter()
+load_dotenv()
+endpoint_logger = setup_logger("endpoint_logger", logging.INFO if os.getenv("GLOBAL_ENVIRONMENT") == "INFO" else logging.DEBUG)
```

Comment:
Same here. If `DEV` then `logging.Debug`. And don't forget about changing the name of the env var